main.py

```diff
@@ -16,15 +16,15 @@ def main():
 
     name = os.environ['NAME']
     key_name = f'{name}-{uuid4()}'
-    launch_template_name = os.environ['LAUNCH_TEMPLATE_NAME']
+    launch_template_names = os.environ['LAUNCH_TEMPLATE_NAMES'].split()
```

Comment:
I think we should split on `,`. Spaces can be difficult to work with in the CLI.

Reply:
good idea, updated 👍